express-notes-api/server.ts

```diff
@@ -26,107 +25,115 @@ app.use(express.json());
 // Returns list of notes in JSON
 app.get('/api/notes', async (req, res) => {
   try {
-    await readJSON();
+    const data = await readJSON();
     const notesList: Note[] = [];
     for (const key in data.notes) {
       notesList.push(data.notes[key]);
     }
     res.status(200).json(notesList);
   } catch (err) {
     console.error(err);
+    error.error = 'An unexpected error occurred.';
```

Comment:
I missed this last time I saw the code. It's strange and I'm not sure what advantage it is giving you over simply:
`res.status(500).json({ error: 'An unexpected error occurred.');`